Makefile

```diff
@@ -700,6 +702,20 @@ generate-images:
 	npm install --no-save --no-package-lock fabric imagemin-zopfli
 	node build/generate-images.js $(TAGS)
 
+.PHONY: generate-manpage
+generate-manpage:
+	@[ -f gitea ] || make backend
+	@mkdir -p man/man1/ man/man5
+	@./gitea docs --man > man/man1/gitea.1
```

Comment:
in usage ...

Reply:
Yep, my bad. I saw the below usage of `go-md2man` and my brain ignored the rest. 😓